pkg/operator/ceph/version/version.go

```diff
@@ -0,0 +1,100 @@
+package version
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+)
+
+type CephVersion struct {
+	Major int
+	Minor int
+	Extra int
+}
+
+// Convenience wrapper combining the Ceph image specification from the cluster
+// CRD along with the verion bits extracted at runtime from the given image.
+type VersionedImage struct {
```

Comment:
Do we need this struct? Seems like after we extract the version from the image, we just need to pass around the `Version` struct